api/routes/otp/schema.js

```diff
@@ -0,0 +1,15 @@
+const postSchema = {
+  body: {
+    type: 'object',
+    required: ['phone'],
+    properties: {
+      phone: {
+        type: 'string',
```

Comment:
if there a validator for this? If not let's just have a regex validator here as well similar to the ORM one.